<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>postinput</class>
 <widget class="QDialog" name="postinput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>投递快递界面</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>461</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_keyboard_postinput">
    <item row="0" column="0">
     <widget class="QPushButton" name="pb1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pb7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pb2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pb3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pb8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pb9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pb5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pb6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pbReturn">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pbDelete">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pb0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter_3_postinput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <layout class="QVBoxLayout" name="verticalLayout_postinput">
     <item>
      <widget class="QSplitter" name="splitter_postinput">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_ID">
        <property name="text">
         <string>快递单号</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leID"/>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_2_postinput">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_Tel">
        <property name="text">
         <string>手机号码</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="leTel"/>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_postinput">
       <item>
        <widget class="QCheckBox" name="cbBig">
         <property name="text">
          <string>大号快递箱</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbMiddle">
         <property name="text">
          <string>中号快递箱</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbSmall">
         <property name="text">
          <string>小号快递箱</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbOK">
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
